modin/pandas/test/test_groupby.py

```diff
@@ -1828,18 +1828,16 @@ def test_multi_column_groupby_different_partitions(
         1.5,
         "str",
         pandas.Timestamp("2020-02-02"),
-        [None],
         [0, "str"],
-        [None, 0],
         [pandas.Timestamp("2020-02-02"), 1.5],
     ],
 )
 @pytest.mark.parametrize("as_index", [True, False])
 def test_not_str_by(by, as_index):
-    data = {f"col{i}": np.arange(5) for i in range(5)}
-    columns = pandas.Index([0, 1.5, "str", pandas.Timestamp("2020-02-02"), None])
+    columns = pandas.Index([0, 1.5, "str", pandas.Timestamp("2020-02-02")])
+    data = {col: np.arange(5) for col in columns}
+    md_df, pd_df = create_test_dfs(data)
 
-    md_df, pd_df = create_test_dfs(data, columns=columns)
```

Comment:
previously the data columns and the keyword columns weren't matching, which caused the dataframes constructor to reindex on the keyword columns that resulted in an empty frame:
```python
>>> data = {"col1": [1, 2], "col2": [3, 4]}
>>> pd.DataFrame(data, columns=["new_name1", "new_name2"])
Empty DataFrame
Columns: [new_name1, new_name2]
Index: []
```